ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/components/properties/BackgroundStyle.kt

```diff
@@ -37,9 +37,9 @@ internal fun Background.toBackgroundStyle(): BackgroundStyle =
             val imageUrls = value.urlsForCurrentTheme
             BackgroundStyle.Image(
                 painter = rememberAsyncImagePainter(
-                    model = imageUrls.webp,
+                    model = imageUrls.webp.toString(),
                     placeholder = rememberAsyncImagePainter(
-                        model = imageUrls.webpLowRes,
+                        model = imageUrls.webpLowRes.toString(),
```

Comment:
This bug was fixed by the tests. It caused the image to not load at all, as we don't have a `Fetcher` that supports the `URL` type.